iron-list.html

```diff
@@ -1058,6 +1063,13 @@
       this.templatize(this._userTemplate, this.mutableData);
     },
 
+    _gridChanged: function(newGrid, oldGrid) {
+      if (typeof oldGrid === 'undefined') return;
+      this.fire('iron-resize');
```

Comment:
Not sure if `.fire` is either correct when also supporting the 2.x ecosystem or worth the overhead when I can  just call `_resizeHandler` directly. Thoughts?